### PR TITLE
Updates imports for org move

### PR DESCRIPTION
**Pull request by bill-rich**

### CODE REVIEW

File: website/docs/r/virtual_machine.html.markdown.orig [hunk 1]
```diff
@@ -0,0 +1,1637 @@
+---
```
Comment (dak1n1) [inline website/docs/r/virtual_machine.html.markdown.orig:1]: Did you mean to include a file that ends with `.orig`?

Comment (bill-rich) [reply website/docs/r/virtual_machine.html.markdown.orig]: I didn't. Thanks for catching that! Got it removed now.

**Comment by bill-rich** [comment]
It was super easy thanks to your doc!

File: vsphere/internal/virtualdevice/virtual_machine_device_subresource.go.orig [hunk 2]
```diff
@@ -0,0 +1,986 @@
+package virtualdevice
```
Comment (dak1n1) [inline vsphere/internal/virtualdevice/virtual_machine_device_subresource.go.orig:1]: Another `.orig` file... I wonder if you meant to include this?

Comment (bill-rich) [reply vsphere/internal/virtualdevice/virtual_machine_device_subresource.go.orig]: Thanks! Looks like that was the last one.

**Pull request merged**